content/bn/event-streaming.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ঘটনা প্রবাহ
+status: Completed
+category: ধারণা
+tags: ["methodology", "networking", ""]
+---
+
+## এটা কি
+
+ঘটনা প্রবাহ হল এমন একটি পদ্ধতি যেখানে সফ্টওয়্যার একটি অ্যাপ্লিকেশন থেকে অন্য অ্যাপ্লিকেশনে ঘটনার তথ্য পাঠায় যাতে তারা কী করছে তা অবিরতভাবে যোগাযোগ করতে পারে।
+অন্য সমস্ত পরিষেবাতে যা করে তা সম্প্রচার করে এমন একটি পরিষেবার চিত্র নিন৷
+একটি পরিষেবা দ্বারা নেওয়া প্রতিটি কার্যকলাপকে একটি ঘটনা হিসাবে উল্লেখ করা হয়, তাই ঘটনা প্রবাহ।
+উদাহরণস্বরূপ, NASDAQ প্রতি সেকেন্ডে স্টক এবং পণ্যের মূল্যের আপডেট পায়।
+আপনার যদি এমন একটি অ্যাপ্লিকেশন থাকে যা স্টকের একটি নির্দিষ্ট সেট নিরীক্ষণ করে, আপনি সেই তথ্যটি কাছাকাছি বর্তমান-সময়ে পেতে চাইবেন।
+ইয়াহু! ফাইন্যান্স একটি [API](/bn/application-programming-interface/) প্রদান করে যা NASDAQ থেকে টেনে নেয় এবং তাদের অ্যাপ্লিকেশন থেকে তথ্য (বা ঘটনা) পাঠায় (বা প্রবাহ) যেকোন অ্যাপ্লিকেশনে যা সদস্যতা নেয়।
+যে তথ্য পাঠানো হচ্ছে সেই সাথে সেই তথ্যের পরিবর্তনগুলি (স্টকের দাম) ঘটনাগুলো যখন একটি অ্যাপ্লিকেশনে সেগুলি বিতরণ করে সেই প্রক্রিয়াটি ঘটনা প্রবাহ।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+ঐতিহ্যগতভাবে, ইয়াহু! ফাইন্যান্স একক TCP অনুরোধ ব্যবহার করবে।
+এটি খুবই অকার্যকর হবে কারণ এটি প্রতিটি ইভেন্টের জন্য একটি সংযোগ তৈরি করতে হয়৷
+তথ্য প্রকৃপক্ষে যত বর্তমান-সময়ে উঠে, এই জাতীয় সমাধানকে স্কেল করা অকার্যকর হয়ে পড়ে।
```

Comment:
Kindly make a clear sentence instead of this sentence ""তথ্য প্রকৃপক্ষে যত বর্তমান-সময়ে উঠে, এই জাতীয় সমাধানকে স্কেল করা অকার্যকর হয়ে পড়ে।"